[Bug] 部署报错

### 💻 Operating System

macOS

### 📦 Environment

Vercel / Zeabur / Sealos

### 🌐 Browser

Chrome

### 🐛 Bug Description

部署报错 
#15 190.3 ./src/hooks/useOAuthSession.ts:1:22

#15 190.3 Type error: Cannot find module '@auth/core/types' or its corresponding type declarations.

#15 190.3 

#15 190.3 > 1 | import { User } from '@auth/core/types';

#15 190.3     |                      ^

#15 190.3   2 | import { SessionContextValue, useSession } from 'next-auth/react';

#15 190.3   3 | import { useMemo } from 'react';

#15 190.3   4 |

#15 190.7  ELIFECYCLE  Command failed with exit code 1.

#15 ERROR: process "/bin/sh -c pnpm run build:docker # run build standalone for docker version" did not complete successfully: exit code: 1

------

 > [builder 6/6] RUN pnpm run build:docker # run build standalone for docker version:

190.2 

190.3 ./src/hooks/useOAuthSession.ts:1:22

190.3 Type error: Cannot find module '@auth/core/types' or its corresponding type declarations.

190.3 

190.3 > 1 | import { User } from '@auth/core/types';

190.3     |                      ^

190.3   2 | import { SessionContextValue, useSession } from 'next-auth/react';

190.3   3 | import { useMemo } from 'react';

190.3   4 |

190.7  ELIFECYCLE  Command failed with exit code 1.

------

Dockerfile:103

--------------------

 101 |     

 102 |     COPY . .

 103 | >>> RUN pnpm run build:docker # run build standalone for docker version

 104 |     

 105 |     ## Production image, copy all the files and run next

--------------------

error: failed to solve: process "/bin/sh -c pnpm run build:docker # run build standalone for docker version" did not complete successfully: exit code: 1

Failed to build with zeaburpack: run buildctl build: exit status 1

### 🚦 Expected Behavior

_No response_

### 📷 Recurrence Steps

_No response_

### 📝 Additional Information

_No response_

--- a/package.json
+++ b/package.json
@@ -82,6 +82,7 @@
   "dependencies": {
     "@ant-design/icons": "^5.3.6",
     "@anthropic-ai/sdk": "^0.18.0",
+    "@auth/core": "0.28.0",
     "@aws-sdk/client-bedrock-runtime": "^3.556.0",
     "@azure/openai": "^1.0.0-beta.12",
     "@cfworker/json-schema": "^1.12.8",

